content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Getting Started with DataTaps in Kubernetes Pods
+date: 2021-07-06T06:44:24.016Z
+author: Cenz Wong
+authorimage: https://avatars.githubusercontent.com/u/44856918?v=4
+tags:
+  - hpe-ezmeral-container-platform
+  - hpe-ezmeral-data-fabric
+---
+## What is DataTap?
+
+(I want to find a better version of this graph showing Kubernetes instead of EPIC, will you guys have that picture? if not, I will create my own.)
+![image](https://user-images.githubusercontent.com/72959956/120766016-62296b00-c54c-11eb-9a2e-6d2ec90e0871.png)
+
+Handling different protocols of file systems is always a pain for a data analyst. DataTap is a file system connector that aims to alleviate this pain. DataTap provides HDFS protocol abstraction that allows big data applications like Spark to run unmodified with fast access to data sources other than HDFS, i.e. HPE Ezmeral Data Fabric XD (formerly named MapR-FS/XD) and GCS (Google Cloud Storage). Using DataTap, you can unify your code while the underlying data sources can be swapped from HDFS, MapR-FS. This flexibility allows developers like you to focus more on coding rather than the underlying infrastructure. More information on DataTap can be found [here](https://docs.containerplatform.hpe.com/53/reference/kubernetes/tenant-project-administration/copy_About_DataTaps.html).
+
+In this blog, I will introduce two ways to access DataTaps in Kubernetes clusters managed by HPE Ezmeral Container Platform deployed with a pre-integrated HPE Ezmeral Data Fabric. The first method covers how to access the DataTaps using HDFS Commands and the second focuses on directly reading data from Apache Spark (using pyspark). Here we go!
+
+
+## **Enable DataTap when creating KubeDirector App**
+
+First and foremost, we have to enable DataTaps while the creation for a KubeDirector app. This can be done by ticking the "Enable DataTap" box.
```

Comment:
Hi, Cenz. This sentence doesn't quite make sense. Is it that you need to enable DataTaps while you are creating a KubeDirector app? Or that you need to enable DataTaps for the creation of a KubeDirector app? Or "while creating a KubeDirector app?"